docs/source/installation.rst

```diff
@@ -4,76 +4,70 @@ Installation
 
 
 Getting the code
-----------------
+================
 
+The recommended way to install Django Glitter is via pip_::
 
-
-The recommended way to install the Debug Toolbar is via pip_::
-    
     $ pip install django-glitter
 
-If you aren't familiar with pip, you may also obtain a copy of the
-``glitter`` directory and add it to your Python path.
-
 .. _pip: http://www.pip-installer.org/
 
 
-Installation
-------------
+settings.py
+===========
 
+INSTALLED_APPS
+--------------
 
-You will need to add the ``glitter`` application to the `INSTALLED_APPS`
-setting of your Django project ``settings.py`` file.::
+You'll need to add ``glitter`` to `INSTALLED_APPS`, as well as any app dependencies, or additional
+blocks which are included with Glitter as Django apps::
 
     INSTALLED_APPS = (
         #...
+
+        # Glitter, and recommended packages
         'glitter',
         'glitter.pages',
-        #...
-    )
-
+        'glitter.assets',
 
-.. note::
+        # Standard blocks
+        'glitter.blocks.html',
+        'glitter.blocks.image',
+        'glitter.blocks.redactor',
 
-    We need to mention here about our required packages ``django-mptt``,
-    ``django-mptt-admin``, ``sorl-thumbnail`` and hint users to add in
-    `INSTALLED_APPS`.
+        # Glitter pages dependencies
+        'mptt',
+        'django_mptt_admin',
+        'sorl.thumbnail',
 
-.. note::
-    
-    Explain diffences between ``glitter`` and ``glitter.pages``
+        #...
+    )
 
```

Comment:
We have to mention that the user has to run `manage.py migrate` after adding required packages into `INSTALLED_APPS`